libs/comments/website/src/lib/comment-editor/comment-editor.tsx

```diff
@@ -1,16 +1,24 @@
 import {zodResolver} from '@hookform/resolvers/zod'
-import {styled} from '@mui/material'
-import {useUser} from '@wepublish/authentication/website'
+import {IconButton, Modal, Theme, css, styled, useTheme} from '@mui/material'
+import {LoginFormContainer, useUser} from '@wepublish/authentication/website'
 import {toPlaintext} from '@wepublish/richtext'
+import {Link} from '@wepublish/ui'
 import {BuilderCommentEditorProps, useWebsiteBuilder} from '@wepublish/website/builder'
-import {useMemo} from 'react'
+import {setCookie} from 'cookies-next'
+import {add} from 'date-fns'
+import {useMemo, useState} from 'react'
 import {Controller, useForm} from 'react-hook-form'
-import {MdSend} from 'react-icons/md'
+import {MdClose, MdLogin, MdSend} from 'react-icons/md'
 import {z} from 'zod'
 
-export const CommentEditorWrapper = styled('form')`
+export const IntendedRouteStorageKey = 'auth.intended'
```

Comment:
Why is this c&p? IntendedRouteStorageKey could be moved to authentication-website and then removed from utils-website.

Reply:
I understand not wanting to copy and paste the same code multiple times, but I don't understand the second part about moving it to authentication-website?

Reply:
How so? Currently it lies within utils-website which you can not use in the website builder since it's for the apps.